MoodDetectorWebApp/Views/Learnings/Learnings.cshtml

```diff
@@ -1,7 +1,84 @@
-﻿
+﻿@using MoodDetectorWebApp.Models
+@using Model;
+@using System.Diagnostics;
+
 @{
     ViewBag.Title = "Learnings";
 }
+<h2>Assigned learnings</h2>
+@if (Model != null)
+{
+    List<Message> learnings = Model;
+    int cnt = 0;
+    for (int i = 0; i < learnings.Count; i++)
+    {
+
+        if (learnings[i].Text == "")
+        {
+            continue;
+        }
+        Debug.WriteLine(learnings[i].Text);
+        cnt++;
+        Debug.WriteLine(learnings[i].SessionTime.ToString());
+        <p>
+            @learnings[i].Text
+            <b id="demo"></b>
+            <script>$("#demo").prop('id', 'demo'+@i)</script >
+            @Html.ActionLink("Link", "RemoveLearning", new { a = learnings[i].SessionId, b = learnings[i].Mask }, new { id = "myLink" + i })
+        </p>
+        <script>
+                $("#myLink"+@i).click(function (event) {
+                    event.preventDefault();
+                    $.ajax({
+
+                        url:$(this).attr("href"),
+
+                    });
+                    $("p").remove(":contains('@learnings[i].Text')")
+
+                });
+        </script>
+        <script>         
+
+            // Update the count down every 1 second
+            var x = setInterval(function () {
+                var ticks = @learnings[i].SessionTime;
+                var ticksToMicrotime = ticks / 10000;
+                var epochMicrotimeDiff = Math.abs(new Date(0, 0, 1).setFullYear(1));
+                var countDownDate = new Date(ticksToMicrotime-epochMicrotimeDiff).getTime();
+                // Get today's date and time
+                var now = new Date().getTime();
 
-<h2>Learnings</h2>
+                // Find the distance between now and the count down date
+                var distance = countDownDate - now;
 
+                // Time calculations for days, hours, minutes and seconds
+                var days = Math.floor(distance / (1000 * 60 * 60 * 24));
+                var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+                var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+                var seconds = Math.floor((distance % (1000 * 60)) / 1000);
+
+                // Output the result in an element with id="demo"
+                document.getElementById('demo'+@i).innerHTML = days + "d " + hours + "h "
+                    + minutes + "m " + seconds + "s ";
+
+                // If the count down is over, write some text
+                if (distance < 0) {
+                    clearInterval(x);
+                    $("#myLink"+@i).trigger("click");
+                }
```

Comment:
So what, when timer is off, you just remove the message? I would suggest just do only days count and write some note when only one days is left, when is overdue etc. It would be nice to show index on menu 'Learnings', how many learnings there are etc.